SuperGraphics.sln

```diff
@@ -13,8 +13,8 @@ Global
 		Release|x86 = Release|x86
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{9F9E9169-2ECB-454C-BB27-28E74B3ED2FE}.Debug|x64.ActiveCfg = Debug|x64
-		{9F9E9169-2ECB-454C-BB27-28E74B3ED2FE}.Debug|x64.Build.0 = Debug|x64
+		{9F9E9169-2ECB-454C-BB27-28E74B3ED2FE}.Debug|x64.ActiveCfg = Debug|Win32
```

Comment:
Please don't commit solution files changes meant for specific platforms!

Reply:
Also for the .DS_Store file in alien texture, remove it and carefully check what you're adding to your commits